back/engines/commercial/multi_tenancy/db/seeds/tenants.rb

```diff
@@ -460,7 +460,8 @@ def create_localhost_tenant
             fake_sso: {
               enabled: true,
               allowed: true,
-              issuer: 'https://fake-sso.onrender.com'
+              # issuer: 'https://fake-sso.onrender.com'
+              issuer: ''
```

Comment:
Don't worry about this, just disabling this for local testing and because we might not use the `onrender.com` deployment after all)

Reply:
Nit: Do you think it could be useful to explain in a comment in which situation we would want to use onrender.com? For example:
```
issuer: '' # You can set this value to 'https://fake-sso.onrender.com' if you want to...
```